doc/nrf/releases_and_maturity/releases/release-notes-2.6.0.rst

```diff
@@ -0,0 +1,1368 @@
+
+* Updated:
+
+  * The :ref:`nrf7002dk_nrf5340` page with a link to the `Wi-Fi Fundamentals course`_ in the `Nordic Developer Academy`_.
+  * The Getting started with nRF70 Series user guide is split into three user guides, :ref:`ug_nrf7002_gs`, :ref:`ug_nrf7002ek_gs` and :ref:`ug_nrf7002eb_gs`.
+  * The Operating with a resource constrained host user guide by renaming it to :ref:`nRF70_nRF5340_constrained_host`.
+    Additionally, the information about stack configuration and performance is placed into its own separate page, :ref:`ug_wifi_stack_configuration`, under :ref:`ug_wifi`.
+
+Working with nRF53 Series
+=========================
+
+* Added the :ref:`qspi_xip` user guide under :ref:`ug_nrf53`.
+
+Working with RF front-end modules
+=================================
+
+* Added a clarification that the Simple GPIO implementation is intended for multiple front-end modules (not just one specific device) in the :ref:`ug_radio_fem_sw_support_mpsl` section of the :ref:`ug_radio_fem` guide.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+See `Samples`_ for lists of changes for the protocol-related samples.
+
+Bluetooth® LE
+-------------
+
+* Added host extensions for Nordic Semiconductor-only vendor-specific command APIs.
+  Implementation and integration of the host APIs can be found in the :file:`host_extensions.h` header file.
+
+Bluetooth Mesh
+--------------
+
+* Added:
+
+  * A script for extracting the Bluetooth Mesh DFU metadata automatically when building an application.
+    The script is enabled by the :kconfig:option:`CONFIG_BT_MESH_DFU_METADATA_ON_BUILD` Kconfig option.
+  * A separate command ``west build -t ble_mesh_dfu_metadata`` to print Bluetooth Mesh DFU metadata that is already generated to a terminal.
+
+* Updated:
+
+  * :ref:`bt_mesh_dm_srv_readme` and :ref:`bt_mesh_dm_cli_readme` model IDs and opcodes to avoid conflict with Simple OnOff Server and Client models.
+  * :ref:`bt_mesh_sensors_readme` to use an updated API with sensor values represented by :c:struct:`bt_mesh_sensor_value` instead of :c:struct:`sensor_value`.
+    This makes it possible to accurately represent all encodable sensor values.
+    The old APIs based on the :c:struct:`sensor_value` type are deprecated, but are still available for backward compatibility and can be enabled for use by setting the :kconfig:option:`CONFIG_BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE` Kconfig option.
+  * The :ref:`bt_mesh_ug_reserved_ids` page with model ID and opcodes for the new :ref:`bt_mesh_le_pair_resp_readme` model.
+  * :ref:`bt_mesh_light_ctrl_readme` APIs to match the new sensor APIs.
+  * The :ref:`ug_bt_mesh_configuring` with the following information:
+
+    * The recommendation on how to configure persistent storage to increase performance.
+    * The required configuration for the network core.
+  * The :ref:`bluetooth_mesh_dfu_eval_md` section with details about the automated metadata generation.
```

Comment:
PR https://github.com/nrfconnect/sdk-nrf/pull/14406 was merged in with a changelog entry to release-notes-changelog.rst. It should be added in this bullet list
```suggestion
  * The :ref:`bluetooth_mesh_dfu_eval_md` section with details about the automated metadata generation.
  * The Kconfig option :kconfig:option:`CONFIG_BT_MESH_NLC_PERF` to no longer select the :kconfig:option:`CONFIG_BT_MESH_MODEL_EXTENSIONS` option.

```

Reply:
Added